WooCommerce/Classes/ViewRelated/Dashboard/MyStore/Cells/ProductTableViewCell.swift

```diff
@@ -61,10 +69,28 @@ extension ProductTableViewCell {
         )
         priceText = statsItem?.formattedTotalString
 
+        /// Set `center` contentMode to not distort the placeholder aspect ratio.
+        /// After a sucessfull image download set the contentMode to `scaleAspectFill`
+        productImage.contentMode = .center
         imageService.downloadAndCacheImageForImageView(productImage,
                                                        with: statsItem?.imageUrl,
                                                        placeholder: .productPlaceholderImage,
-                                                       progressBlock: nil,
-                                                       completion: nil)
+                                                       progressBlock: nil) { [productImage] (image, _) in
```

Comment:
I just noticed that this uses strong capturing. Which is weird because we're using `productImage?` below. Can we explicitly mention `weak`? 

```suggestion
                                                       progressBlock: nil) { [weak productImage] (image, _) in
```

Reply:
Oh, 🤦‍♂

Reply:
Fixed https://github.com/woocommerce/woocommerce-ios/pull/2004/commits/58678968a26493f8f9618668b6bb2ab05119b46c